Dockerfile

```diff
@@ -7,6 +7,9 @@ COPY logdna.gpg /etc/
 RUN echo "deb http://repo.logdna.com stable main" > /etc/apt/sources.list.d/logdna.list && \
     apt-key add /etc/logdna.gpg && \
     apt-get -y update && \
-    apt-get -y install logdna-agent
-
+    apt-get -y install logdna-agent && \
+    apt-get -y upgrade apt && \
```

Comment:
Can you remove the `apt` and see if the image size stay the same? you are still restricting it here.

it should be `apt-get -y upgrade && \`

Reply:
Upgrading `apt` and `systemd` don't seem like ideal and permanent solution - what have we agreed on regarding using `alpine`? Staying on `ubuntu:16.04`?

Reply:
ubuntu updates their image monthly.  running updates in the image is generally good practice.

that being said I'd like for us to consider using the alpine image.

Reply:
with `apt-get -y upgrade` the image size is same as specifying `apt` and `systemd`. 
This is a temp fix, we need to move to alpine.